Apps/Database/src/Migrations/20210618175942_AddInAppCommunicationType.cs

```diff
@@ -0,0 +1,43 @@
+﻿// -------------------------------------------------------------------------
```

Comment:
Can you add the PushBannerChange trigger to this migration and update it to include the InApp type?

Reply:
Good call, i forgot about that trigger